src/app.js

```diff
@@ -330,5 +302,18 @@ export default function (app) {
 
     return element
   }
+
+  function event(type) {
+    var args = Array.prototype.slice.call(arguments, 1);
+    (subscriptions[type] || []).forEach(function(cb) { cb.apply(null, args) });
+    (subscriptions["*"] || []).forEach(function(cb) { cb.apply(null, type) });
```

Comment:
It was part of Mitt, as I said above. If we get a Chrome Dev tool for Hyperapp in the future it will be useful. Since that's a maybe, we can remove it for now and add when needed.